pkg/cli/cli.go

```diff
@@ -205,7 +207,7 @@ func (o *stdOS) Readline(opts interp.ReadlineOpts) (string, error) {
 
 func (o *stdOS) History() ([]string, error) {
 	// TODO: refactor history handling to use internal fs?
-	r, err := os.Open(o.rl.Config.HistoryFile)
+	r, err := os.Open(o.historyFile)
```

Comment:
Now seeing the TODO above i remember that it would be nice if readline had a history API that didn't use `os` directly. Would be more flexible and easier to test.